src/opened.ts

```diff
@@ -42,21 +51,32 @@ async function opened() {
 
   // Query for ticket page object
   const [_, username, ticketNumStr, ticketName] = matches;
-  const ticket = await getTicket(parseInt(ticketNumStr));
-
-  if (ticket === null) {
-    setFailed(`No ticket found with ID ${ticketNumStr}`);
-    return;
+  const ticketNum = parseInt(ticketNumStr);
+
+  // Get/create ticket
+  // Fails if it can't find the specified ticket
+  let ticket: TicketPage;
+  if (ticketNumStr === "") {
+    const fetchedTicket = await getTicket(ticketNum);
```

Comment:
I'm confused, how does ticketNum get set?

I might be understanding wrong this would mean a ticket WOULDN'T get created in notion?